pom.xml

```diff
@@ -530,6 +533,7 @@
                         <exclude>docs/demo/**/*.ipynb</exclude>
                         <exclude>docs/demo/**/*.zpln</exclude>
                         <exclude>**/src/main/resources/META-INF/services/*</exclude>
+                        <exclude>tests-next-spark/target/**/*</exclude>
```

Comment:
What I have found is that rat disables checking of the target directory for all active projects.  If the project is not active (i.e. it was disabled by a profile) the rat will check it and likely fail if it was not cleaned up in between.  I have been bitten by this a few times.  I personally am fine with this for all sub-projects that we enable/disable, but I would almost rather have it be `**/target/**/*` along with an explanation as to why we are doing this (because it is not obvious). Or just remove the rule because it is not hard to delete a directory when you make a mistake.